plugin/wavesurfer.regions.js

```diff
@@ -182,6 +184,36 @@ WaveSurfer.Region = {
         }
 
         this.updateRender();
+
+        var regionEl = document.querySelector('[data-id="'+this.id+'"]');
+
+        /* Resize handles */
+        if (this.resize && !regionEl.firstChild) {
+            var handleLeft = regionEl.appendChild(document.createElement('handle'));
+            var handleRight = regionEl.appendChild(document.createElement('handle'));
+            handleLeft.className = 'wavesurfer-handle wavesurfer-handle-start';
+            handleRight.className = 'wavesurfer-handle wavesurfer-handle-end';
+            var css = {
+                cursor: 'col-resize',
+                position: 'absolute',
+                left: '0px',
+                top: '0px',
+                width: '1%',
+                maxWidth: '4px',
+                height: '100%'
+            };
+            this.style(handleLeft, css);
+            this.style(handleRight, css);
+            this.style(handleRight, {
+                left: '100%'
+            });
+            this.element = this.wrapper.appendChild(regionEl);
+        }
+
+        if(!currentDrag || !currentResize){
+            this.bindEvents(regionEl);
+        }
```

Comment:
I think it's better to extract this part of the [`render` function](https://github.com/katspaugh/wavesurfer.js/blob/master/plugin/wavesurfer.regions.js#L213-L257) into a separate function and call it from both places passing either a freshly created region element or an existing element.